android-svc-lib.sh

```diff
@@ -370,6 +455,15 @@ AidlSyntaxHighlight () {
     echo "${l_highlightedCode}"
 }
 
+###
+# Set binary name to call to access android device.
+# get first android serial.
+# GLOBALS:
+#  g_shellType is modified to contains android serial id
```

Comment:
`g_shellType` is not modified. If `g_adbSerial` has not been set before, the script will try to auto detect the id. (not exactly sure what I did there with sed at the top of my mind)